src/main/resources/META-INF/spring/app-context.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:context="http://www.springframework.org/schema/context"
+       xmlns:mongo="http://www.springframework.org/schema/data/mongo"
+       xmlns:util="http://www.springframework.org/schema/util"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans			http://www.springframework.org/schema/beans/spring-beans.xsd
+                        http://www.springframework.org/schema/context		http://www.springframework.org/schema/context/spring-context-3.0.xsd
+                        http://www.springframework.org/schema/data/mongo    http://www.springframework.org/schema/data/mongo/spring-mongo-1.0.xsd
+                        http://www.springframework.org/schema/util          http://www.springframework.org/schema/util/spring-util-3.0.xsd">
+
+
+    <context:component-scan base-package="com.feedbacksolution" />
+
+    <!-- Host and port for the mongod instance (default values of MongoDB here) -->
+    <mongo:mongo host="localhost" port="27017" />
```

Comment:
this can also go to config file and we can use java annotations to get access to it